api/managementpb/backup/artifacts.proto

```diff
@@ -18,6 +18,7 @@ enum BackupStatus {
   BACKUP_STATUS_ERROR = 5;
   BACKUP_STATUS_DELETING = 6;
   BACKUP_STATUS_FAILED_TO_DELETE = 7;
```

Comment:
How about this?
```suggestion
  BACKUP_STATUS_DELETION_FAILED = 7;
```

Reply:
Replied in original comment https://github.com/percona/pmm/pull/1894#discussion_r1173623022